drape_frontend/visual_params.cpp

```diff
@@ -224,12 +224,12 @@ m2::RectD GetRectForDrawScale(int drawScale, m2::PointD const & center)
 
 m2::RectD GetRectForDrawScale(double drawScale, m2::PointD const & center, uint32_t tileSize, double visualScale)
 {
-  return GetRectForDrawScale(base::SignedRound(drawScale), center, tileSize, visualScale);
+  return GetRectForDrawScale(static_cast<int>(std::lround(drawScale)), center, tileSize, visualScale);
```

Comment:
Hm, we don't need special casts for the cases below (uint8_t, uint16_t, int) but why need here?

Reply:
`long` returned by `lround` is ambiguous when casting to `GetRectForDrawScale` accepting `int` or `double`.